sched_slack_bot/utils/fix_schedule_from_the_past.py

```diff
@@ -12,7 +12,7 @@ def fix_schedule_from_the_past(schedule: Schedule) -> Schedule:
     logger.info(f"Starting to fix schedule with previous date {schedule.next_rotation}")
 
     while now >= next_rotation:
-        next_rotation += schedule.time_between_rotations
+        next_rotation += abs(schedule.time_between_rotations)
```

Comment:
I don't think this is the only place to change, to fix this, but I am not sure ;)

Reply:
gotcha :) I think there was only one more, where it was saved to the db

Reply:
Do we still need this after you fixed `time_between_rotations = abs(second_rotation - next_rotation)`?